CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## master
 
+### Fixes
+
+* `[jest-runtime]` Replace path sep for testRegex in should_instrument so test
+files don't get instrumented on windows given a pattern like /__tests__/*.js
```

Comment:
you need to stick /\_\_tests\_\_/ in backticks, if not markdown looks really weird